dojo/finding/views.py

```diff
@@ -433,15 +433,15 @@ def close_finding(request, fid):
             if len(missing_note_types) == 0:
                 finding.active = False
                 now = timezone.now()
-                finding.mitigated = now
-                finding.mitigated_by = request.user
+                finding.mitigated = form.cleaned_data["mitigated"] or now
+                finding.mitigated_by = form.cleaned_data["mitigated_by"] or request.user
                 finding.is_mitigated = True
                 finding.last_reviewed = finding.mitigated
                 finding.last_reviewed_by = request.user
                 endpoint_status = finding.endpoint_status.all()
                 for status in endpoint_status:
-                    status.mitigated_by = request.user
-                    status.mitigated_time = timezone.now()
+                    status.mitigated_by = form.cleaned_data["mitigated_by"] or request.user
+                    status.mitigated_time = form.cleaned_data["mitigated"] or now
```

Comment:
Would be better to use the safer accessor instead: `form.cleaned_data.get("mitigated_by", None)`